ganttproject-tester/test/biz/ganttproject/core/calendar/WeekendCalendarImplTest.java

```diff
@@ -56,15 +56,19 @@ public DateFormat getShortDateFormat() {
       }
     };
   }
-  private static List<CalendarEvent> TEST_EVENTS = ImmutableList.of(
+
+  private static List<CalendarEvent> createTestEvents() {
+    return ImmutableList.of(
       CalendarEvent.newEvent(CalendarFactory.createGanttCalendar(2014, 0, 1).getTime(), true, CalendarEvent.Type.HOLIDAY, "Jan 1", null),
       CalendarEvent.newEvent(CalendarFactory.createGanttCalendar(2014, 1, 14).getTime(), false, CalendarEvent.Type.NEUTRAL, "Feb 14", null),
       CalendarEvent.newEvent(CalendarFactory.createGanttCalendar(2014, 2, 8).getTime(), true, CalendarEvent.Type.HOLIDAY, "Mar 8", null),
       CalendarEvent.newEvent(CalendarFactory.createGanttCalendar(2014, 2, 8).getTime(), false, CalendarEvent.Type.WORKING_DAY, "Mar 8, 2014", null),
       CalendarEvent.newEvent(CalendarFactory.createGanttCalendar(2014, 2, 9).getTime(), false, CalendarEvent.Type.HOLIDAY, "Mar 9, 2014", null),
       CalendarEvent.newEvent(CalendarFactory.createGanttCalendar(2014, 3, 12).getTime(), true, CalendarEvent.Type.WORKING_DAY, "Apr 12", null),
       CalendarEvent.newEvent(CalendarFactory.createGanttCalendar(2014, 3, 12).getTime(), false, CalendarEvent.Type.HOLIDAY, "Apr 12, 2014", null)
-  );
+    );
+  }
+  private static List<CalendarEvent> TEST_EVENTS = createTestEvents();
```

Comment:
Could you please explain the trick? :)
Why to create `TEST_EVENTS` and use it in the first method only?
Shouldn't `TEST_EVENTS_RECURRING_FIRST` be converted to method as well for consistency?

Reply:
Thanks, I replaced both static lists with the method calls.
The issue which I struggled with was that these tests were failing if I ran them all (like `gradle test`). However, with a filter (even a very simple `gradle test --tests biz*`) they were passing. 
I didn't get the reasons exactly, but I think that there were side effects of setting the static member `ourLocaleApi` from different tests. That's why I replaced static instances and static setup with instances created per each test.